west.yml

```diff
@@ -120,15 +120,15 @@ manifest:
     - name: wfa-qt-control-app
       repo-path: sdk-wi-fiquicktrack-controlappc
       path: modules/lib/wfa-qt-control-app
-      revision: 0e213ab6a421298484a81a9d18f5c31c4b8084a8
+      revision: vX.Y.Z-ncsN-snapshot1
```

Comment:
Need version details for wfa-qt-control-app

Reply:
Same here, do we really need to rebase that repo?

Reply:
I don't think so as seldom there are updates and there is none.